mkt/ratings/models.py

```diff
@@ -120,6 +130,47 @@ def transformer(reviews):
         for user in UserProfile.objects.no_cache().filter(id__in=user_ids):
             user_ids[user.id].user = user
 
+    def guess_language(self):
```

Comment:
The language guessing code feels a bit out of place in `mkt.ratings.models`. Maybe it should go in a utils module somewhere?